www/src/components/CSSUtilitiesTable.tsx

```diff
@@ -1,8 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+// @ts-ignore
 import { Table } from '~paragon-react'; // eslint-disable-line
 
-function CSSUtilitiesTable({ selectors, showExample }) {
+export type CSSUtilitiesTableTypes = {
+  selectors: [{
+    selector: string,
+    declarations: Array<string>,
+  }],
+  showExample: Array<string>,
```

Comment:
[curious] Per the prop type definitions below, `showExample` expects a boolean true/false, not an Array<string>?

Reply:
Yes, you are right, thanks